### PR TITLE
Fix/hooked problems

### LINKED ISSUE #762: [Eurovision] Countdown to playback jumpy
## Description of the problem

The 3 sec countdown before playing a song goes down, up and down again.

## Expected behaviour

...

## How to reproduce

- Experiment/ruleset: Eurovision experiment, 
- Environment: local develop, acceptance server, 
- Browsers: Firefox, Chromium

### LINKED ISSUE #1018: 🐛 [BUG] - Kuiper and Eurovision: first trial does not work properly
### Description

For both the Kuiper and Eurovision rules, the first round is not displayed properly, it does not play audio and goes straight to the (empty) score screen.

### Screenshots
Screenshot 2024-05-21 at 13 56 52

### Reproduction URL

https://illc-aml-mu-prd.science.uva.nl/christmas_2020?participant_id=test

### Reproduction steps

```bash
1. Start experiment
2. Second screen after intro / consent will be faulty.
```

### Logs

```bash
The network log shows two consecutive `next_round` calls. Is the `first_round` array problematic?
```

### Browsers

_No response_

### OS

_No response_

### Additional context

_No response_

### LINKED ISSUE #1022: 🐛 [BUG] - Hooked: "Silence phase" shown when clicking "No" for the "recognize" question.
### Description

This is probably only noticeable at slow connection speeds: the "Silence phase" ("Keep the music playing in your head") is briefly shown due to an `await` before the condition which continues to the score page.

### Reproduction URL

_No response_

### Reproduction steps

```bash
1. Throttle connection speed through dev tools
2. Start a "Hooked" game, click "No" for the "Do you recognize the song?" question.
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by BeritJanssen**
Close #1018, close #762 (again), close #1022.

### CODE REVIEW

**Comment by Evert-R** [comment]
The issues in this PR are solved but now when running matching pairs or categorization the questionaire isn't shown and the experiment freezes. No error message just an empty screen.

no-questions

**Review by Evert-R — CHANGES_REQUESTED** [review]
The issues in this PR are solved but now when running matching pairs or categorization the questionaire isn't shown and the experiment freezes. No error message just an empty screen.

no-questions

**Comment by BeritJanssen** [comment]
Ah, thanks for catching that! Removing the `await` message seemed to be without side effects for the experiments I tested, but I guess it's the culprit for breaking MatchingPairs and Categorization.

File: frontend/src/components/Trial/Trial.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Trial = ({
 }) => {
     // Main component state
     const [formActive, setFormActive] = useState(!config.listen_first);
-    const [preloadReady, setPreloadReady] = useState(!(playback?.ready_time));
+    const [preloadReady, setPreloadReady] = useState(false);
```
Comment (BeritJanssen) [inline frontend/src/components/Trial/Trial.jsx:26]: It's actually this part which is problematic, as we don't have a `setPreloadReady` command in Trials without Playback.

**Pull request merged**